subjects/btreedeletenode/main.go

```diff
@@ -4,6 +4,11 @@ import (
 	"fmt"
 )
 
+type TreeNode struct {
```

Comment:
Shouldn't we avoid the definition of the `TreeNode` type and assume the student will define it?

Reply:
In the piscine the have the TreeNode defined in the firsts exercises of the quest but in the exam they wont have access to the piscine so we need to give the TreeNode so that they can test the code in the code editor

Reply:
Removed the TreeNode and added the functions they need to test with the main.go